client/layout/index.js

```diff
@@ -101,9 +101,24 @@ export class PageLayout extends Component {
 			<Router history={ getHistory() }>
 				<Switch>
 					{ getPages().map( page => {
-						return <Route key={ page.path } path={ page.path } exact component={ Layout } />;
+						return (
+							<Route
+								key={ page.path }
+								path={ page.path }
+								exact
+								render={ props => (
+									<Layout
+										breadcrumbs={
+											( isFunction( page.breadcrumbs )
+												? page.breadcrumbs( props )
+												: page.breadcrumbs ) || []
+										}
+										{ ...props }
+									/>
+								) }
+							/>
+						);
 					} ) }
-					<Route component={ Layout } />
```

Comment:
This "default" route shouldn't exist. Is there any reason for it?

Reply:
This should be the 404 page